packages/amplify-cli-core/src/serviceSelection.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsdoc/require-jsdoc */
```

Comment:
I'd disable this rule as well.

Reply:
Agree, but same thoughts as above, I would like to disable in a follow up PR